src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -239,6 +239,18 @@
     <value>Prefer 'static readonly' fields over constant array arguments if the called method is called repeatedly and is not mutating the passed array</value>
     <comment>{Locked="static readonly"}</comment>
   </data>
+  <data name="AvoidUnreliableStreamReadTitle" xml:space="preserve">
+    <value>Avoid unreliable call to 'Stream.Read'</value>
```

Comment:
For me the word `unreliable` is a bit harsh, could be something like:
```suggestion
    <value>Avoid inexact read with 'Stream.Read'</value>
```
But if others are OK with it, then fine, tag @gewarren @stephentoub for opinions

Reply:
I like your suggestion.

Reply:
Thanks, I've added the new title and message in 254f3a868989ada9bc710a0d873e0efd5b5331a7.